src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1101,6 +1221,102 @@ components:
+        page:
+          type: integer
+          description: The page number
+          example: 1
+        pageSize:
+          type: integer
+          description: The number of results per page
+          example: 1
+        totalPages:
+          type: integer
+          description: The total number of pages
+          example: 1
+        totalResults:
+          type: integer
+          description: The total number of results
+          example: 1
+        results:
+          type: array
+          description: Array of anonymous application search summaries that matches the search criteria
+          items:
+            $ref: '#/components/schemas/ApplicationSearchSummaryResponse'
+      required:
+        - page
+        - pageSize
+        - totalPages
+        - totalResults
+        - results
+
+    ApplicationSearchSummaryResponse:
+      title: ApplicationSearchSummaryResponse
+      description: A response containing summary information of an each Anonymous Elector document application
+      type: object
+      properties:
+        gssCode:
+          type: string
+          description: The relevant GSS code for the local authority.
+          example: 'E09000007'
+        sourceReference:
+          type: string
+          pattern: '^[a-fA-F\d]{24}$'
+          description: |
+            A 24 character hex string. 
+            The unique identifier of an application which should be used as the resource ID for all operations on that Anonymous elector document application.
+          example: 1f0f76a9a66f438b9bb33070
+        applicationReference:
+          type: string
+          description: The reference number of an application
+          example: A3JSZC4CRH
+        certificateNumber:
+          type: string
+          description: The unique identifier of the Anonymous Electoral Document. A 20 character alphanumeric string.
+          pattern: '^[0-9AC-HJ-NP-RT-Z]{20}$'
+          example: DEV1W0XDH1368LWE1J40
+        electoralRollNumber:
+          type: string
+          description: Electoral roll number printed on the AED. There is no specific format and they can vary across EROs.
+          maxLength: 25
+        firstName:
+          type: string
+          description: The applicant's first name
+          example: Fred
+        surname:
+          type: string
+          description: The applicant's surname
+          example: Blogs
+        postcode:
+          type: string
+          description: The applicant's postcode
+          example: AA1 1AA
+        issueDate:
+          type: string
+          format: date
+          description: The date on which the AED was issued
+          example: '2023-01-15'
+        dateTimeCreated:
+          type: string
+          format: date-time
+          description: The timestamp in ISO-8601 UTC that the application document was originally created
+          example: '2022-07-27T09:39:44Z'
+      required:
+        - gssCode
+        - sourceReference
```

Comment:
We could also return `source_type` if required. What do you think? Not sure its value though

Reply:
Personally I don't see the need.